2022-07_sprint/sprint_parties/contributors.md

```diff
@@ -400,11 +400,12 @@ Bhopal, Madhya Pradesh, India
 :::
 :::{grid-item}
 
-<img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
+<img alt="Image for contributor template" src="../../_static/contributors_2022_07/Margaret_Okore.JPG" class="rounded-circle">
 
-Margaret Chege
+Margaret Okore
 
-Kenya
+Nairobi, Kenya
+[{fab}`linkedin-in`](https://www.linkedin.com/in/margaretokore/)
```

Comment:
```suggestion
Nairobi, Kenya

[{fab}`linkedin-in`](https://www.linkedin.com/in/margaretokore/)
```

Reply:
I believe there needs to be a blank line.

Reply:
I concluded the blank line